nixos/tests/mysql.nix

```diff
@@ -28,6 +28,12 @@ import ./make-test.nix ({ pkgs, ...} : {
       {
         users.users.testuser = { };
         services.mysql.enable = true;
+        services.mysql.initialScript = pkgs.writeText "mariadb-init.sql" ''
```

Comment:
I think we should add support for secured installations as an option instead of doing it here, if the current state is terribly insecure. It shouldn't be too complicated, and in general will make these setups more predictable.

Reply:
@flokli well this is merely a test to ensure someone doesn't make this mistake again. I also would like our `mysql` module to either have a secure option of just come secure by default, but that is for another PR.